WapTenantPublicAPI.psm1

```diff
@@ -549,6 +549,192 @@ function GetWAPSubscriptionQuota {
     
 }
 
+function Get-WAPSubscriptionQuota{
```

Comment:
I did notice that. It seems to only be designed to work with SQL servers. I haven't got any of those yet so I can't outright test the function. 

The internal function simply returns basequotasettings but only for SQL servers. It assumes you've selected a subscription. 

My one collects subscription.service.basequotasettings.clouds.cloud.quota as well as calculating the current consumption. It takes Subscription Objects as inputs. 
 
So they both have radically different views on what we are getting the Quota for (one is SQL, one is VM provisioning). 

Possible options: 
1. Expand the WAP.Quota object to include SQL so that if it's a SQL subscription, it includes the relavant parts. 

2. Rename them to make the names clearer - perhaps Get-WAPSQLQuota and Get-WAPSubscriptionCloudQuota

Since the original helper function is never exported, we can be confident that renaming the old one won't dramatically break anything :)